### ISSUE TITLE
The output of volume rendering and rasterization on Neural_3D_Video Dataset

### ISSUE THREAD

**Issue by ronly2460** [TQTQliu/MVSGaussian#62]
I tried the Neural_3D_Video dataset on a pretrained model, but the output of the rasterization was poor, while the volume rendering results look nice.

Do you have any ideas about this?

Dataset: https://github.com/facebookresearch/Neural_3D_Video

### Volume rendering
![flame_steak_5_0](https://github.com/user-attachments/assets/cd7f02dd-c12c-4bf7-b348-ba35490f9b5d)

### Rasterization
![flame_steak_5_0](https://github.com/user-attachments/assets/c132ce0a-209b-4bf6-af19-279cfd5bd4c9)

**Comment by TQTQliu** [comment]
Hi, thanks for your interest.

This is probably due to `scale_factor`. Since our model is trained on DTU data set (depth range is 425~905), when testing on other new datasets, we use scale_factor to adjust the depth of the new scenario to be close to the depth of the DTU dataset. For example, for the nerf synthetic dataset with a depth range of 2.5 to 5.5, we set scale_factor to 200, and for the LLFF dataset with a depth range of about 20 to 100, we set it to 12. You would try modifying scale_factor [here](https://github.com/TQTQliu/MVSGaussian/blob/2efbe9d51c67fb4e4b7c945265948e611751692a/configs/mvsgs/colmap_eval.yaml#L9).
